### ISSUE TITLE
Error downloading from Google Drive

### ISSUE THREAD

**Issue by NoUsernameThanks** [Js41637/Overwatch-Item-Tracker#203]
Made the mistake of not moving my local backup out of my download folder before taking a break from the game, now it's been deleted and my last backup is 2 events old, aside from whats stored on google drive. But trying to get that i just get the error message:

![image](https://user-images.githubusercontent.com/34490620/38661214-5bede816-3e30-11e8-9e0f-909d4ae41234.png)

I've recreated it in chrome, Firefox, IE 11 and Edge, and tried it without privacy blockers etc. There's nothing logged in my console either.

**Comment by Js41637** [comment]
Yeah Google Drive stuff is fucked, I dunno what's wrong, it's not really designed for what I'm trying to do. It needs a lot of work.

**Comment by NoUsernameThanks** [comment]
So, probably best I just power on and suck it up, i assume?

**Comment by Js41637** [comment]
Uhh, I might try some stuff, the problem with Google Drive is its super hard to know whats stored in there and users (such as yourself (or me)) don't have any clue what is actually stored in there.

It probably wont be quick though.

**Comment by NoUsernameThanks** [comment]
Alright, I'll just stick to hoarding boxes for now, thanks!

**Comment by tarix** [comment]
When you have no local storage data at all then service.dataFileID (services.js) is null and so the whole thing quickly fails. I tried hacking getData so if the service.dataFileID is null it would use findDataFile to set it and it seems to download the data but then I get the error "Data returned by Google Drive does not matched expected data".
